index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <h1 id="title">freeCodeCamp Survey Form</h1>
+  <p id="description">freeCodeCamp Survey Form</p>
+  <form id="survey-form">
```

Comment:
It'd be great if the form has a test action to check if it is submitting the information correctly.